src/components/test/NavElements.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import configureStore from '../../redux/configureStore';
+import '@testing-library/jest-dom/extend-expect';
+import NavElement from '../NavElement/NavElement';
+
+describe('Tests the Header component', () => {
+  const component = (
+    <BrowserRouter>
+      <Provider store={configureStore}>
+        <NavElement />
+      </Provider>
+    </BrowserRouter>
+  );
+
+  test('Match the snapshot', () => {
+    const tree = render(component);
+    expect(tree).toMatchSnapshot();
```

Comment:
- Your test is failing here. kindly fix it.

![image](https://github.com/cancelei/Stock-Exchange-APP/assets/59397159/e636d846-f36f-4baf-a1b4-f26fc21a5c13)

Reply:
![image](https://github.com/cancelei/Stock-Exchange-APP/assets/96393104/cc8bdaa3-1a17-4b1d-b878-58aeff260ddc)